### PR TITLE
Reintroduce cashbox

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Cashbox PT1: Re-introduce the components on the React app](https://trello.com/c/CN8bJjwY/423-cashbox-pt1-re-introduce-the-components-on-the-react-app)

**Comment by chaotixkilla** [comment]
[Cashbox PT2: Make the cash-in transactions update the total cashbox count as well](https://trello.com/c/OS45Doio/424-cashbox-pt2-make-the-cash-in-transactions-update-the-total-cashbox-count-as-well)

File: lib/cash-in/cash-in-atomic.js [hunk 1]
```diff
@@ -46,8 +46,15 @@ function insertNewBills (t, billRows, machineTx) {
   const dbBills = _.map(cashInLow.massage, bills)
   const columns = _.keys(dbBills[0])
   const sql = pgp.helpers.insert(dbBills, columns, 'bills')
-
-  return t.none(sql)
+  const deviceID = machineTx.deviceId
+  const sql2 = `update devices set cashbox = cashbox + 1
```
Comment (RafaelTaranto) [inline lib/cash-in/cash-in-atomic.js]: Are we sure this is only one at a time? Can you try shutting down the server after the first bill insertion -> insert a couple more bills -> turn the server back on.

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: lib/cash-in/cash-in-atomic.js (hunk 1 above)
Comment (chaotixkilla) [reply lib/cash-in/cash-in-atomic.js]: I was thinking, wouldn't it be cleaner to add dbBills.length? Would solve that problem and would be more understandable, as it is not a number spawned out of nowhere

File: new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js [hunk 2]
```diff
@@ -17,6 +17,8 @@ import styles from './FiatBalanceAlerts.styles.js'
 const useStyles = makeStyles(styles)
 
 const NAME = 'fiatBalanceAlerts'
+// const CASH_IN_KEY = 'fiatBalanceAlertsCashIn'
```
Comment (hellypi) [inline new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js]: Remove annotations from file (unless it's temporary?)

File: new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.styles.js [hunk 3]
```diff
@@ -2,6 +2,10 @@ export default {
   wrapper: {
     display: 'flex'
   },
+  /* formWrapper: {
```
Comment (hellypi) [inline new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.styles.js]: same here

File: new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceOverrides.js [hunk 4]
```diff
@@ -9,6 +9,7 @@ import Autocomplete from 'src/components/inputs/formik/Autocomplete'
 import NotificationsCtx from '../NotificationsContext'
 import { transformNumber } from '../helper'
 
+// const CASHBOX_KEY = 'cashbox'
```
Comment (hellypi) [inline new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceOverrides.js]: same here

**Review by hellypi — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js (hunk 2 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js]: Yes, it is all temporary. Stuff in this PR which is commented is to be uncommented later. Resolving all conversations related to this.

**Review by RafaelTaranto — APPROVED** [review]

File: new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js [hunk 5]
```diff
@@ -32,6 +34,12 @@ const FiatBalance = ({
   const editing = isEditing(NAME)
 
   const schema = Yup.object().shape({
+    /* cashInAlertThreshold: Yup.number()
```
Comment (joshmh) [inline new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js]: Can we delete all the commented out code in this PR?

**Review by joshmh — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js (hunk 5 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js]: Should be all removed by now. Future PRs introducing the cashbox to the alerts and balance override, should be able to reuse the commented code

**Pull request merged**